[Docs] Add documentation for KontextInpaintingPipeline (#12197)
huggingface/diffusers: docs/source/en/api/pipelines/flux.md

    raise ValueError("Your image has been flagged. Choose another prompt/image or try again.")
```

### Kontext Inpainting

> The Kontext Inpainting Pipeline enables image modification within a fixed mask region. It currently supports both text-based conditioning and image-reference conditioning.

Reviewer: ```suggestion
[`FluxKontextInpaintPipeline`] enables image modification within a fixed mask region. It currently supports both text-based conditioning and image-reference conditioning.
```

> #### Inpainting with text only

Reviewer: Wrap both these sections with `<hfoptions>`:

```md
<hfoptions id="kontext-inpaint">
<hfoption id="text-only">

code goes here

</hfoption>
<hfoption id="image conditioning">

code goes here

</hfoption>
</hfoptions>

    raise ValueError("Your image has been flagged. Choose another prompt/image or try again.")
```

### Kontext Inpainting
`FluxKontextInpaintPipeline` enables image modification within a fixed mask region. It currently supports both text-based conditioning and image-reference conditioning.
<hfoptions id="kontext-inpaint">
<hfoption id="text-only">

> #### Inpainting with text only

Reviewer: ```suggestion
```

```python
import torch
from diffusers import FluxKontextInpaintPipeline
from diffusers.utils import load_image

prompt = "Change the yellow dinosaur to green one"
img_url = (
    "https://github.com/ZenAI-Vietnam/Flux-Kontext-pipelines/blob/main/assets/dinosaur_input.jpeg?raw=true"
)
mask_url = (
    "https://github.com/ZenAI-Vietnam/Flux-Kontext-pipelines/blob/main/assets/dinosaur_mask.png?raw=true"
)

source = load_image(img_url)
mask = load_image(mask_url)

pipe = FluxKontextInpaintPipeline.from_pretrained(
    "black-forest-labs/FLUX.1-Kontext-dev", torch_dtype=torch.bfloat16
)
pipe.to("cuda")

image = pipe(prompt=prompt, image=source, mask_image=mask, strength=1.0).images[0]
image.save("kontext_inpainting_normal.png")
```
</hfoption>
<hfoption id="image conditioning">

> #### Inpainting with image conditioning

Reviewer: ```suggestion
```
